src/database.ts

```diff
@@ -427,6 +430,7 @@ class Database extends common.GrpcServiceObject {
       Object.assign({}, queryOptions),
       Database.getEnvironmentQueryOptions()
     );
+    this.loggingEnabled = false;
```

Comment:
This is set false by default here but will get set as true when loggingEnabled function is called from the session pool constructor. If logging is false in the session pool options, then this will remain to be false